### PR TITLE
Review styles - #149

### PR DESCRIPTION
**Pull request by irina060981**
https://github.com/alpheios-project/components/issues/149

1) I have added normalize and ui_base styling - that creates base styling inside popup and panel, to be sure outside styles won't rewrite it

2) I have made little fixes to font and color styling (to remove unused constants)

3) I have fixed updating classes to avoid removing observing feature.

### CODE REVIEW

File: src/styles/_alpheios_backup.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+$alpheios-uikit-namespace: 'auk';
```
Comment (kirlat) [inline src/styles/_alpheios_backup.scss]: Do we need a backup file? If its purpose is to save an old state of things, we have VCS for that

Comment (irina060981) [reply src/styles/_alpheios_backup.scss]: oh, you are right - I will remove all old files.

File: src/styles/ui_base.scss [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+  .alpheios-skin_properties_item__large_font {
+  	font-size: $alpheios-base-font-size * 1.2;
+  	width: $alpheios-base-font-size * 2.1;
+  	height: $alpheios-base-font-size * 2.1;
+	line-height: $alpheios-base-font-size * 2.1;
+	border-radius: $alpheios-base-font-size * 1.1;
+  }
+
+  .alpheios-grammar__frame-cont {
```
Comment (balmas) [inline src/styles/ui_base.scss]: Why add grammar styles here? Is there a reason not to specify them fully in  grammar.vue?

**Comment by irina060981** [comment]
I agree with you, @balmas, I have missed this idea :)
I have rearranged some styles from uibase to vue-components, multiselect and checkbox.

**Pull request merged**